docs/modules/ROOT/pages/python-client-getting-started.adoc

```diff
@@ -0,0 +1,211 @@
+
+== Use Map
+
+A Hazelcast map is a distributed key-value store, similar to Python Map structure. In the following example, we will work with map entries where the keys are session ids and the values are emails.
+
+[source, python]
+----
+import hazelcast
+
+client = hazelcast.HazelcastClient()
+
+some_map = client.get_map("some-map")
+
+some_map.set("12345", "example1@email.com")
+some_map.set("12346", "example2@email.com")
+
+entries = some_map.entry_set().result()
+
+for key, value in entries:
+    print("%s -> %s" % (key, value))
+
+client.shutdown()
+----
+
+
+The output snippet is
+[source, bash]
+----
+example1@email.com
+example2@email.com
+----
+
+Since Hazelcast clients work asynchronous, our output is not in order.
+
+Following line returns a map proxy object for the 'some-map' named map.
+
+[source, python]
+some_map = client.get_map("some-map")
+
+If there is not a map named ”some-map”, it will be automatically created. All the clients connected to the same cluster will have access to same map.
+
+With these lines, client adds data to our “some-map”. The first parameter is the key of the entry, the second one is the value as mentioned above.
+
+[source, python]
+----
+some_map.set("12345", "example1@email.com")
+some_map.set("12346", "example2@email.com")
+----
+
+Then, we get the the values with the entry_set().result() method. You may wonder why we have used result()method over the entry_set(). This is because the Hazelcast Python client is designed to be fully asynchronous. Every method call over distributed objects such as put(), get(), entry_set(), etc. will return a https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Future[Future] object that is similar to the Future class of the https://docs.python.org/3/library/concurrent.futures.html#module-concurrent.futures[concurrent.futures] module.
+
+With this design choice, method calls over the distributed objects can be executed asynchronously without blocking the execution order of your program.
+
+Finally, client.shutdown() shutdowns our client.
+
+== Add a Listener to the Map
+
+You can add entry listener using the add_entry_listener() method available on map proxy object. This will allow you to listen to certain events that happen in the map across the cluster.
+
+The first argument to add_entry_listener() method is includeValue. It is a boolean parameter, and if it is true, the entry event contains the entry value. In this example, it will be true.
+
+The second argument to add_entry_listener() method is an object that is used to define listeners. In this example, we registered listeners for the “added”, “removed” and “updated” events.
+
+[source, python]
+----
+import hazelcast
+
+def entry_added(event):
+    print("Entry added with key: %s, value: %s" % (event.key, event.value))
```

Comment:
Let's use f-strings instead of `%` formatting.

Reply:
Done.